Project.toml

```diff
@@ -3,7 +3,10 @@ uuid = "d76558cf-badf-52d4-a17e-381ab0b0d937"
 version = "0.1.5"
 
 [deps]
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
```

Comment:
Could you remove this unnecessary dependencies?

Reply:
oops, leftover from other testing codes I have to compare the parsers in julia python and matlab, gonna remove that indeed.

Reply:
I removed HTTP as well then, since it's now useless I think